### PR TITLE
Teams endpoint

### PR DESCRIPTION
**Pull request by tanmaydattta**
(untested) registration/teams endpoints

### CODE REVIEW

File: fattmerchant/Merchant.py [hunk 1]
```diff
@@ -281,3 +282,336 @@ def set_merchant_id(self, id):
...
+            "team_role": "full",
+            "team_enabled": true,
+            "send_verification_email": true,
+            "url": "https://omni.fattmerchant.com/#/verify/"
+            }
+        """
+
+        pass
+    
+    def team_create_api_key_user(self, name:str, role:str):
```
Comment (AustinBurns) [inline fattmerchant/Merchant.py]: Remove typing for 2.7.

Comment (AustinBurns) [inline fattmerchant/Merchant.py]: I think we can just name this `team_create_user` and denote in the doc string that it also returns an API key for the user.

File: fattmerchant/Merchant.py [hunk 2]
```diff
@@ -281,3 +282,336 @@ def set_merchant_id(self, id):
...
+            "team_enabled": true,
+            "send_verification_email": true,
+            "url": "https://omni.fattmerchant.com/#/verify/"
+            }
+        """
+
+        pass
+    
+    def team_create_api_key_user(self, name:str, role:str): 
+        """creates api key
```
Comment (AustinBurns) [inline fattmerchant/Merchant.py]: 1. Pull the method description up to here

2. Place the note below the description.

**Comment by AustinBurns** [comment]
So I noticed that the doc string was inconsistent with some of the methods so let's figure out a consistent doc string structure and then fix all of the doc strings in this PR to adhere to that doc string structure. Slack me whenever you want to go over this.

File: fattmerchant/Merchant.py [hunk 3]
```diff
@@ -281,3 +282,336 @@ def set_merchant_id(self, id):
         fattmerchant client to interact with api 
         """
         self.id = id
+
+    def team_create(self, team_info:dict):
```
[the remark this answers is not in the document]
Comment (tanmaydattta) [reply fattmerchant/Merchant.py]: It is already done in the 2.7 specific branch

File: fattmerchant/Merchant.py [hunk 4]
```diff
@@ -281,3 +282,336 @@ def set_merchant_id(self, id):
...
+            "contact_phone": "8555503288",
+            "address_1": "25 Wall Street",
+            "address_2": "Suite 1",
+            "address_city": "Orlando",
+            "address_state": "FL",
+            "address_zip": "32801"
+            }
+
+        """
+        return 1
```
[the remark this answers is not in the document]
Comment (tanmaydattta) [reply fattmerchant/Merchant.py]: Looks like this is outdated ?

File: fattmerchant/Merchant.py [hunk 5]
```diff
@@ -281,3 +282,336 @@ def set_merchant_id(self, id):
...
+
+    def team_edit_info(self, team_info:dict):
+        """some summary
```
[the remark this answers is not in the document]
Comment (tanmaydattta) [reply fattmerchant/Merchant.py]: we are going to merge 2.7 into master, please reivew that instead.

**Pull request closed without merging**